src/modules/new-request-form/translations/en-us.yml

```diff
@@ -6,6 +6,7 @@
 # If you are building your own theme, you can remove this file and just load the translation JSON files, or provide
 # your translations with a different method.
 #
+
```

Comment:
We can revert this change that just adds an empty line

Reply:
Changed